### ISSUE TITLE
Marking too many messages as spam at once causes "internal error"

### ISSUE THREAD

**Issue by skandragon** [modoboa/modoboa-amavis#76]
When marking around 50 messages as spam at once, it causes an error to be reported to the UI.

**Comment by skandragon** [comment]
Nope, that's it.  I suspect it's when you select too many messages and a command line argument is exceeded.

**Comment by fyfe** [comment]
The next release of modoboa that includes modoboa/modoboa#1336 should help to find the cause. When enabled it'll email the admin a python stack trace which should point out what's causing this.

**Comment by skandragon** [comment]
Awesome, I'll upgrade once it's out.  Thanks!
